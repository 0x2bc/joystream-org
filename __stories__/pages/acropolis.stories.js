import React from 'react';
import { storiesOf } from '@storybook/react';

import AcropolisPage from '../../src/pages/acropolis';

storiesOf('Pages|Acropolis', module).add('default', () => <AcropolisPage />);
