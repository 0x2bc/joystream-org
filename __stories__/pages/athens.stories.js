import React from 'react';
import { storiesOf } from '@storybook/react';

import AthensPage from '../../src/pages/athens';

storiesOf('Pages|Athens', module).add('default', () => <AthensPage />);
