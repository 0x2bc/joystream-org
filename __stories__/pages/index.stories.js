import React from 'react';
import { storiesOf } from '@storybook/react';

import IndexPage from '../../src/pages/index';

storiesOf('Pages|Home', module).add('default', () => <IndexPage />);
