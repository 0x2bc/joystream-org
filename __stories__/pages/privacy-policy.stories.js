import React from 'react';
import { storiesOf } from '@storybook/react';

import PrivacyPolicyPage from '../../src/pages/privacy-policy';

storiesOf('Pages|Privacy Policy', module).add('default', () => <PrivacyPolicyPage />);
