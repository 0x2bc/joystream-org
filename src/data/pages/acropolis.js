const goalsData = [
  {
    title: 'Build and release an on-chain forum',
    text: `The final platform will have built in multiple ways of facilitating
            easy and secure public, and private, communication and information
            sharing among members. The forum is the first step allowing the
            platform members to communicate, share ideas and discuss.`,
  },
  {
    title: 'Rebuild and release the storage node',
    text: `Where the old storage node had some bugs making it unable to sync
            between clients and required a hardcoded liaison, the new storage
            node is working as intended with no hierarchy or privileges.`,
  },
  {
    title: 'Ensure high uptime of storage providers',
    text: `Both content creators and consumers user experience depends on
            various metrics, but perhaps the most important is that the storage
            node they connect to responds to their request. Whether or not this
            goal is achieved depends on both the reliability of the software, a
            good reporting system and corrective actions from multiple parties.`,
    state: 'inprogress',
  },
  {
    title: 'Build the storage node with "tranches"',
    text: `Although the content directory is currently small, at some point
            it's not feasible to expect all storage providers to have full
            replication of all the content. Tranches would allow storage
            providers to select a subset of the content directory, suitable to
            their capabilities.`,
    state: 'postponed',
  },
];

export { goalsData };
