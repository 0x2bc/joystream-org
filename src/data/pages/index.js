import { ReactComponent as ValidatorsImage } from '../../assets/svg/active-validators.svg';
import { ReactComponent as StorageImage } from '../../assets/svg/storage.svg';
import { ReactComponent as MemberImage } from '../../assets/svg/council-member.svg';
import { ReactComponent as ScreenerImage } from '../../assets/svg/screener.svg';
import { ReactComponent as MembershipCuratorImage } from '../../assets/svg/membership-curator.svg';
import { ReactComponent as ContentCuratorImage } from '../../assets/svg/content-curator.svg';
import { ReactComponent as ContentCreatorImage } from '../../assets/svg/content-creator.svg';
import { ReactComponent as BandwidthProviderImage } from '../../assets/svg/bandwidth-provider.svg';
import { ReactComponent as DiscoveryProviderImage } from '../../assets/svg/discovery-provider.svg';
import { ReactComponent as LiveStreamingProviderImage } from '../../assets/svg/live-streaming-provider.svg';
import { ReactComponent as BuilderImage } from '../../assets/svg/builder.svg';
import { ReactComponent as CommunicationModeratorImage } from '../../assets/svg/communication-moderator.svg';

const roles = {
  active: [
    {
      image: ValidatorsImage,
      title: 'Validator',
      to: '/roles#Validator',
      key: 'validatorsCount',
      hasLabel: false,
    },
    {
      image: MemberImage,
      title: 'Council Member',
      to: '/roles#Council-Member',
      key: 'councilMembersCount',
      hasLabel: false,
    },
    {
      image: StorageImage,
      title: 'Storage Provider',
      to: '/roles#Storage-Provider',
      key: 'storageProviders',
      hasLabel: true,
    },
  ],
  future: [
    { image: ScreenerImage, title: 'Membership Screener', to: '/roles' },
    {
      image: MembershipCuratorImage,
      title: 'Membership Curator',
      to: '/roles',
    },
    { image: ContentCuratorImage, title: 'Content Curator', to: '/roles' },
    { image: ContentCreatorImage, title: 'Content Creator', to: '/roles' },
    {
      image: BandwidthProviderImage,
      title: 'Bandwidth Provider',
      to: '/roles',
    },
    {
      image: DiscoveryProviderImage,
      title: 'Discovery Provider',
      to: '/roles',
    },
    {
      image: LiveStreamingProviderImage,
      title: 'Live Streaming Provider',
      to: '/roles',
    },
    { image: BuilderImage, title: 'Builder', to: '/roles' },
    {
      image: CommunicationModeratorImage,
      title: 'Communication Moderator',
      to: '/roles',
    },
  ],
};

export { roles };
