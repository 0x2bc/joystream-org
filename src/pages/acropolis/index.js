import React, { useState } from 'react';
import { string, shape, number } from 'prop-types';

import getApiPath from '../../utils/getApiPath';
import mapStatusDataToAnalytics from '../../utils/mapStatusDataToAnalytics';

import withApi from '../../components/_enhancers/withApi';

import BaseLayout from '../../components/_layouts/Base';
import HeroCard from '../../components/HeroCard';
import Analytics from '../../components/Analytics';
import TitleWrapper from '../../components/TitleWrapper';
import RoleCard from '../../components/RoleCard';
import ColumnsLayout from '../../components/ColumnsLayout';
import LayoutWrapper from '../../components/LayoutWrapper';
import Hero from '../../components/Hero';
import TestnetModal from '../../components/TestnetModal';
import Chip from '../../components/Chip';
import Pane from '../../components/Pane';
import GoalItem from '../../components/GoalItem';
import Link from '../../components/Link';

import { ReactComponent as AcropolisImage } from '../../assets/svg/acropolis.svg';
import { ReactComponent as Info } from '../../assets/svg/info.svg';
import { ReactComponent as SpecImg } from '../../assets/svg/specifications.svg';
import { ReactComponent as ReleaseImg } from '../../assets/svg/release-doc.svg';
import AcropolisImg from '../../assets/svg/acropolis.svg';

import { roles } from '../../data/pages';

import './style.scss';
import MapInfo from '../../components/MapInfo';
import { ReactComponent as PersonIcon } from '../../assets/svg/person.svg';

const propTypes = {
  content: shape({
    block_height: number,
    council: shape({
      election_stage: string,
      members_count: number,
    }),
    forum: shape({
      posts: number,
      threads: number,
    }),
    media: shape({
      media_files: number,
    }),
    memberships: shape({
      platform_members: number,
    }),
    roles: shape({
      storage_providers: number,
    }),
    runtime_version: shape({
      impl_name: string,
      spec_name: string,
      spec_version: number,
    }),
    system: shape({
      chain: string,
      name: string,
      peerCount: number,
      version: string,
    }),
    validators: shape({
      count: number,
      total_stake: string,
    }),
  }).isRequired,
};

const AcropolisPage = ({ content }) => {
  const [isModalOpen, setModalClosed] = useState(false);

  return (
    <BaseLayout>
      <Hero
        image={AcropolisImage}
        title="Acropolis Network"
        indent
        chip={
          <Chip onClick={() => setModalClosed(true)} icon={Info}>
            What is this?
          </Chip>
        }
      >
        <p className="AcropolisPage__hero-paragraph">
          Explore available roles and pick the one that suits you the most.
          Influence platforms development earning Monero in the process.
        </p>
        <HeroCard date="2019/06/27 17:50" />

        <TestnetModal
          title="The Acropolis of Athens"
          image={AcropolisImg}
          closeModal={() => setModalClosed(false)}
          isOpen={isModalOpen}
        >
          <p>
            <strong>Known for its great architecture, Acropolis'</strong>{' '}
            perhaps most famous building is the Parthenon. It was built to
            celebrate their victory over Persian invaders, and is today seen as
            a symbol for democracy and western civilization.
          </p>
        </TestnetModal>
      </Hero>

      <LayoutWrapper>
        <TitleWrapper title="Network Statistics">
          <Analytics content={mapStatusDataToAnalytics(content)} />
        </TitleWrapper>
      </LayoutWrapper>

      <LayoutWrapper>
        <TitleWrapper title="Critical Documents">
          <ColumnsLayout>
            <Pane
              image={SpecImg}
              href="https://github.com/Joystream/joystream/tree/master/testnets/acropolis/specification"
              title="Full Specifications"
              target="_blank"
            >
              Read the specs of the newly implemented features of Acropolis.
            </Pane>
            <Pane
              image={ReleaseImg}
              href="https://github.com/Joystream/joystream/tree/master/testnets/acropolis"
              title="Release Plan"
              target="_blank"
            >
              Read the release plan as it was made during the planning stage,
              and learn more about how the development evolved.
            </Pane>
          </ColumnsLayout>
        </TitleWrapper>
      </LayoutWrapper>

      <LayoutWrapper>
        <TitleWrapper
          title="Testnet Goals"
          subtitle={
            <>
              The goals below is a simplified representation of the Key Results
              listed in our Release{' '}
              <Link href="https://github.com/Joystream/joystream/tree/master/testnets/acropolis#release-okrs">
                OKR
              </Link>
            </>
          }
        >
          <GoalItem title="Build and release an on-chain forum">
            The final platform will have built in multiple ways of facilitating
            easy and secure public, and private, communication and information
            sharing among members. The forum is the first step allowing the
            platform members to communicate, share ideas and discuss.
          </GoalItem>

          <GoalItem title="Rebuild and release the storage node">
            Where the old storage node had some bugs making it unable to sync
            between clients and required a hardcoded liaison, the new storage
            node is working as intended with no hierarchy or privileges.
          </GoalItem>

          <GoalItem
            state="inprogress"
            title="Ensure high uptime of storage providers"
          >
            Both content creators and consumers user experience depends on
            various metrics, but perhaps the most important is that the storage
            node they connect to responds to their request. Whether or not this
            goal is achieved depends on both the reliability of the software, a
            good reporting system and corrective actions from multiple parties.
          </GoalItem>

          <GoalItem
            state="postponed"
            title='Build the storage node with "tranches"'
          >
            Although the content directory is currently small, at some point
            it's not feasible to expect all storage providers to have full
            replication of all the content. Tranches would allow storage
            providers to select a subset of the content directory, suitable to
            their capabilities.
          </GoalItem>
        </TitleWrapper>
      </LayoutWrapper>

      <LayoutWrapper dark>
        <TitleWrapper title="Roles available on the current testnet">
          <ColumnsLayout>
            {roles.active.map(role => (
              <RoleCard {...role} key={role.title} />
            ))}
          </ColumnsLayout>
        </TitleWrapper>
      </LayoutWrapper>

      <MapInfo title="Acropolis of Athens" location="acropolis">
        <p>
          <strong>
            The Acropolis is a citadel on a hill in the heart of Athens.
          </strong>{' '}
          It was also at the heart of Ancient Greece, a powerful civilization
          and empire. Acropolis is famous for its ancient buildings,
          architecture, historical significance and is one of the main tourist
          attractions of Athens. It is on UNESCOs list "World Heritage Sites".
          <br />
          <br />
          We chose the name as we had to scale back our ambitions for the next
          testnet after some issues with the release of Athens. Thus, Acropolis
          can be considered a sub-release, despite including some new features
          not intended for Athens.
          <br />
          <br />
          <Link to="/athens">
            <PersonIcon /> Explore previous testnet
          </Link>
        </p>
      </MapInfo>
    </BaseLayout>
  );
};

AcropolisPage.propTypes = propTypes;

export { AcropolisPage };
export default withApi(AcropolisPage, getApiPath('STATUS'));
