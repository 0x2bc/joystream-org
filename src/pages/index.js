import React from 'react';
import { pagePropTypes } from '../propTypes';

import getApiPath from '../utils/getApiPath';
import mapStatusDataToAnalytics from '../utils/mapStatusDataToAnalytics';
import mapStatusDataToRoles from '../utils/mapStatusDataToRoles';

import withApi from '../components/_enhancers/withApi';

import BaseLayout from '../components/_layouts/Base';
import Link from '../components/Link';
import Subheader from '../components/Subheader';
import TestnetItem from '../components/TestnetItem';
import Analytics from '../components/Analytics';
import Button from '../components/Button';
import TitleWrapper from '../components/TitleWrapper';
import RoleList from '../components/RoleList';
import ColumnsLayout from '../components/ColumnsLayout';
import LayoutWrapper from '../components/LayoutWrapper';
import Hero from '../components/Hero';
import SiteMetadata from '../components/SiteMetadata';

import AcropolisImage from '../assets/svg/acropolis-main.svg';
import heroImage from '../assets/svg/hero-builder.svg';
import RomeImage from '../assets/svg/rome-main.svg';
import constantinopleImage from '../assets/svg/constantinople-main.svg';
import alexandriaImage from '../assets/svg/alexandria-main.svg';
import { ReactComponent as TickImage } from '../assets/svg/tick.svg';
import { ReactComponent as ClockImage } from '../assets/svg/clock.svg';

import { roles } from '../data/pages';
import { launchDate as alexandriaNetworkLaunchDate } from '../data/pages/alexandria';

import './style.scss';

const activeTestnet = {
  name: 'Alexandria',
  incentivesLink: 'https://www.joystream.org/testnet',
};

const IndexPage = ({ content }) => (
  <BaseLayout>
    <SiteMetadata
      title="Joystream: The video platform DAO"
      description="Joystream is a video platform controlled, owned, and operated by its users."
    />
    <Hero
      image={heroImage}
      title={
        <>
          The video <br /> platform DAO
        </>
      }
      animationStartValue={0}
    >
      <p className="IndexPage__hero-paragraph">
        Joystream is a video platform controlled, owned, and operated by its users.
      </p>
      <div className="IndexPage__hero-group">
        <Button noWrap large secondary className="IndexPage__hero-button" href="/testnet/">
          Earn Tokens
        </Button>
        <Button
          noWrap
          large
          secondary
          reversed
          className="IndexPage__hero-button"
          href="https://testnet.joystream.org/"
        >
          Launch UI
        </Button>
      </div>
    </Hero>

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children={
          <>Alexandria is our seventh testnet, introducing a variety of important technical improvements.</>
        }
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    <Analytics title="Testnet Metrics" large content={mapStatusDataToAnalytics(content)}>
      <Button secondary href={activeTestnet.incentivesLink}>
        Participate and Earn Monero
      </Button>
    </Analytics>

    {/*

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children="Alexandria is our seventh testnet, focused on a variety of important technical improvements."
        date={alexandriaNetworkLaunchDate}
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    */}

    <LayoutWrapper dark>
      <TitleWrapper title="Become a user, run the platform">
        <Subheader
          title="Active roles on the current testnet"
          content="learn more, join a role and subscribe for more"
          icon={TickImage}
        />
        <ColumnsLayout>
          <RoleList roles={roles.active} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>
        <Subheader
          title="Roles coming in future testnets"
          content="choose a role, learn more and subscribe to get updated"
          icon={ClockImage}
        />
        <ColumnsLayout columnsCount={3}>
          <RoleList roles={roles.future} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>

        <Button secondary to="/roles" className="IndexPage__roles-button">
          Learn more about the roles
        </Button>
      </TitleWrapper>
    </LayoutWrapper>
  </BaseLayout>
);

IndexPage.propTypes = pagePropTypes;

export { IndexPage };
export default withApi(IndexPage, getApiPath('/'));
