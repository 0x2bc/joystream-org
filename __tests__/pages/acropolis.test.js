import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import { AcropolisPage } from '../../src/pages/acropolis';
import testContent from '../../__mocks__/data/testContent';

describe('AcropolisPage page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AcropolisPage content={testContent} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
