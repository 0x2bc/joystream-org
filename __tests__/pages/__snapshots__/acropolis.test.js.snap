// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AcropolisPage page renders correctly 1`] = `
<div>
  <nav
    className="Navbar"
  >
    <div
      className="Navbar__container"
    >
      <a
        className="Link"
        href="/"
      >
        <icon-mock
          className="Navbar__logo"
        />
      </a>
      <div
        className="Navbar__links"
      >
        <a
          className="Navbar__link Navbar__link--highlighted Link Link--highlighted"
          href="https://www.jsgenesis.com/#openings"
          label="We are hiring!"
        >
          We are hiring!
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/"
          label="Blog"
        >
          Blog
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/manifesto/"
          label="Manifesto"
        >
          Manifesto
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/whitepaper/blob/master/paper.pdf"
          label="Whitepaper"
        >
          Whitepaper
        </a>
        <a
          className="Navbar__link Link"
          href="/roles"
          label="Roles"
        >
          Roles
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/joystream"
          label="Repository"
        >
          Repository
        </a>
      </div>
      <div
        className="Navbar__trigger"
        onClick={[Function]}
        role="presentation"
      />
    </div>
  </nav>
  <section
    className="Hero Hero--indented"
  >
    <div
      className="Hero__container"
    >
      <div
        className="Hero__content"
      >
        <h1
          className="Hero__title"
        >
          Acropolis Network
        </h1>
        <div
          className="Hero__description"
        >
          <p
            className="AcropolisPage__hero-paragraph"
          >
            Explore available roles and pick the one that suits you the most. Influence platforms development earning Monero in the process.
          </p>
          <div
            className="Card"
          >
            <p
              className="Card__header"
            >
              <icon-mock
                className="Card__icon"
              />
              Network announced
            </p>
            <div
              className="Card__content"
            >
              <div
                className="DateCounter DateCounter--light DateCounter--large"
              >
                <div
                  className="DateCounter__title"
                >
                  Launched on
                </div>
                <div
                  className="DateCounter__container"
                >
                  <div
                    className="DateCounter__segment"
                  >
                    <span
                      className="DateCounter__value"
                    >
                      27
                    </span>
                    <span
                      className="DateCounter__label"
                    >
                      day
                    </span>
                  </div>
                  <span
                    className="DateCounter__value, DateCounter__separator"
                  >
                    /
                  </span>
                  <div
                    className="DateCounter__segment"
                  >
                    <span
                      className="DateCounter__value"
                    >
                      06
                    </span>
                    <span
                      className="DateCounter__label"
                    >
                      month
                    </span>
                  </div>
                  <span
                    className="DateCounter__value, DateCounter__separator"
                  >
                    /
                  </span>
                  <div
                    className="DateCounter__segment"
                  >
                    <span
                      className="DateCounter__value"
                    >
                      19
                    </span>
                    <span
                      className="DateCounter__label"
                    >
                      year
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="Hero__image-wrapper"
      >
        <button
          className="Chip"
          onClick={[Function]}
        >
          <icon-mock
            className="Chip__icon"
          />
          <span
            className="Chip__text"
          >
            What is this?
          </span>
        </button>
        <div
          className="Plx  "
          disabled={false}
          style={
            Object {
              "visibility": "hidden",
            }
          }
        >
          <icon-mock
            className="Hero__image"
          />
        </div>
      </div>
    </div>
  </section>
  <div
    className="LayoutWrapper"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Network Statistics
        </h2>
        
        <section
          className="Analytics"
        >
          <div
            className="Analytics__container"
          >
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Participation Payout
              </p>
              <p
                className="AnalyticsItem__value"
              >
                $1576
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Active Validators
              </p>
              <p
                className="AnalyticsItem__value"
              >
                20
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Block Height
              </p>
              <p
                className="AnalyticsItem__value"
              >
                1110117
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Council Election Stage
              </p>
              <p
                className="AnalyticsItem__value"
              >
                Announcing
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Memberships
              </p>
              <p
                className="AnalyticsItem__value"
              >
                164
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Forum Posts
              </p>
              <p
                className="AnalyticsItem__value"
              >
                11
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Platform Content Files
              </p>
              <p
                className="AnalyticsItem__value"
              >
                9
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Active Storage Providers
              </p>
              <p
                className="AnalyticsItem__value"
              >
                10
              </p>
            </div>
          </div>
        </section>
      </section>
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Critical Documents
        </h2>
        
        <div
          className="ColumnsLayout"
          data-columns={2}
        >
          <a
            className="Pane Link"
            href="https://github.com/Joystream/joystream/tree/master/testnets/acropolis/specification"
            target="_blank"
          >
            <div
              className="Pane__wrapper"
            >
              <icon-mock
                className="Pane__img"
              />
            </div>
            <div
              className="Pane__content"
            >
              <p
                className="Pane__title"
              >
                Full Specifications
              </p>
              <p
                className="Pane__paragraph"
              >
                Read the specs of the newly implemented features of Acropolis.
              </p>
            </div>
          </a>
          <a
            className="Pane Link"
            href="https://github.com/Joystream/joystream/tree/master/testnets/acropolis"
            target="_blank"
          >
            <div
              className="Pane__wrapper"
            >
              <icon-mock
                className="Pane__img"
              />
            </div>
            <div
              className="Pane__content"
            >
              <p
                className="Pane__title"
              >
                Release Plan
              </p>
              <p
                className="Pane__paragraph"
              >
                Read the release plan as it was made during the planning stage, and learn more about how the development evolved.
              </p>
            </div>
          </a>
        </div>
      </section>
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Testnet Goals
        </h2>
        <div
          className="TitleWrapper__subtitle"
        >
          The goals below is a simplified representation of the Key Results listed in our Release
           
          <a
            className="Link"
            href="https://github.com/Joystream/joystream/tree/master/testnets/acropolis#release-okrs"
          >
            OKR
          </a>
        </div>
        <div
          className="GoalItem"
        >
          <div
            className="GoalItem__icon-container"
          >
            <icon-mock
              className="GoalItem__icon"
            />
            <p
              className="GoalItem__status GoalItem__status--achieved"
            >
              Achieved
            </p>
          </div>
          <div
            className="GoalItem__content"
          >
            <p
              className="GoalItem__title"
            >
              Build and release an on-chain forum
            </p>
            The final platform will have built in multiple ways of facilitating
            easy and secure public, and private, communication and information
            sharing among members. The forum is the first step allowing the
            platform members to communicate, share ideas and discuss.
          </div>
        </div>
        <div
          className="GoalItem"
        >
          <div
            className="GoalItem__icon-container"
          >
            <icon-mock
              className="GoalItem__icon"
            />
            <p
              className="GoalItem__status GoalItem__status--achieved"
            >
              Achieved
            </p>
          </div>
          <div
            className="GoalItem__content"
          >
            <p
              className="GoalItem__title"
            >
              Rebuild and release the storage node
            </p>
            Where the old storage node had some bugs making it unable to sync
            between clients and required a hardcoded liaison, the new storage
            node is working as intended with no hierarchy or privileges.
          </div>
        </div>
        <div
          className="GoalItem"
        >
          <div
            className="GoalItem__icon-container"
          >
            <icon-mock
              className="GoalItem__icon"
            />
            <p
              className="GoalItem__status GoalItem__status--inprogress"
            >
              In Progress
            </p>
          </div>
          <div
            className="GoalItem__content"
          >
            <p
              className="GoalItem__title"
            >
              Ensure high uptime of storage providers
            </p>
            Both content creators and consumers user experience depends on
            various metrics, but perhaps the most important is that the storage
            node they connect to responds to their request. Whether or not this
            goal is achieved depends on both the reliability of the software, a
            good reporting system and corrective actions from multiple parties.
          </div>
        </div>
        <div
          className="GoalItem"
        >
          <div
            className="GoalItem__icon-container"
          >
            <icon-mock
              className="GoalItem__icon"
            />
            <p
              className="GoalItem__status GoalItem__status--postponed"
            >
              Postponed
            </p>
          </div>
          <div
            className="GoalItem__content"
          >
            <p
              className="GoalItem__title"
            >
              Build the storage node with "tranches"
            </p>
            Although the content directory is currently small, at some point
            it's not feasible to expect all storage providers to have full
            replication of all the content. Tranches would allow storage
            providers to select a subset of the content directory, suitable to
            their capabilities.
          </div>
        </div>
      </section>
    </div>
  </div>
  <div
    className="LayoutWrapper LayoutWrapper--dark"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Roles available on the current testnet
        </h2>
        
        <div
          className="ColumnsLayout"
          data-columns={2}
        >
          <a
            className="RoleCard Link"
            href="/roles#Validator"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Validator
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                <span
                  className="RoleCard__counter"
                >
                  20
                </span>
                 currently run this role
              </p>
            </div>
          </a>
          <a
            className="RoleCard Link"
            href="/roles#Council-Member"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Council Member
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                <span
                  className="RoleCard__counter"
                >
                  12
                </span>
                 currently run this role
              </p>
            </div>
          </a>
          <a
            className="RoleCard RoleCard--labeled Link"
            href="/roles#Storage-Provider"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Storage Provider
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                <span
                  className="RoleCard__counter"
                >
                  10
                </span>
                 currently run this role
              </p>
            </div>
          </a>
        </div>
      </section>
    </div>
  </div>
  <section
    className="MapInfo"
  >
    <div
      className="Map MapInfo__map"
      style={
        Object {
          "backgroundImage": "url(test-file-stub)",
        }
      }
    >
      <icon-mock />
      <p
        className="Map__info"
      >
        Map data ©
        2019
         Google
      </p>
    </div>
    <div
      className="MapInfo__content"
    >
      <h1
        className="MapInfo__title"
      >
        Acropolis of Athens
      </h1>
      <p>
        <strong>
          The Acropolis is a citadel on a hill in the heart of Athens.
        </strong>
         
        It was also at the heart of Ancient Greece, a powerful civilization and empire. Acropolis is famous for its ancient buildings, architecture, historical significance and is one of the main tourist attractions of Athens. It is on UNESCOs list "World Heritage Sites".
        <br />
        <br />
        We chose the name as we had to scale back our ambitions for the next testnet after some issues with the release of Athens. Thus, Acropolis can be considered a sub-release, despite including some new features not intended for Athens.
        <br />
        <br />
        <a
          className="Link"
          href="/athens"
        >
          <icon-mock />
           Explore previous testnet
        </a>
      </p>
    </div>
  </section>
  <footer
    className="Footer"
  >
    <p
      className="Footer__header"
    >
      Get notified
    </p>
    <a
      className="Footer__button Button Button--small Button--light"
      href="http://joystream.us11.list-manage.com/subscribe/post?u=932de577aec9616d4516b4e0f&id=459ba8d1da"
      onClick={null}
    >
      Join the newsletter
    </a>
    <div
      className="Footer__socials"
    >
      <a
        className="Footer__social-wrapper"
        href="https://twitter.com/JoyStreamApp"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://github.com/Joystream"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://chat.joystream.org/home"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://t.me/joinchat/CNyeUxHD9H56m3e_44hXIA"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
    </div>
    <div
      className="Footer__links"
    >
      <a
        className="Footer__link Footer__link--highlighted Link Link--highlighted"
        href="https://www.jsgenesis.com/#openings"
        label="We are hiring!"
      >
        We are hiring!
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/"
        label="Blog"
      >
        Blog
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/manifesto/"
        label="Manifesto"
      >
        Manifesto
      </a>
      <a
        className="Footer__link Link"
        href="https://github.com/Joystream/joystream"
        label="Repository"
      >
        Repository
      </a>
    </div>
  </footer>
</div>
`;
